README.md

```diff
@@ -171,21 +171,20 @@ Feature: Richdeps/Behave test-1
 Scenario: Install TestA from repository "test-1"
  Given I use the repository "test-1"
  When I execute "dnf" command "install -y TestA" with "success"
- Then package "TestA, TestB" should be "installed"
- And package "TestC" should be "absent"
+ Then transaction changes are as follows
+   | State        | Packages      |
+   | installed    | TestA, TestC  |
+   | absent       | TestB         |
 
 ```
 
 Possible actions:
-    for rpm: install, remove
-    for dnf: install, remove, upgrade, upgrade (with package name: all),
-    upbrade-to, downgrade, autoremove
-    for command-line commands: unlimited + dnf-2 and dnf-3 run
+    for bash commands: unlimited + dnf-2 and dnf-3 runs
 
 Possible package managers: dnf, rpm
 
-Possible states: installed, removed, absent, upgraded, unupgraded, downgraded,
-present, upgraded-to
+Possible states: installed, removed, absent, upgraded, downgraded,
+present
```

Comment:
document that states `absent` and `present` are optional in definitions of tests and/or in the step definition itself, please.